Description: Workaround environment variable settings
 Add environment to be set by packaging
Forwarded: not-needed
Author: Colin Ian King <colin.i.king@gmail.com>

Index: intel-cmt-cat-23.11/lib/Makefile
===================================================================
--- intel-cmt-cat-23.11.orig/lib/Makefile
+++ intel-cmt-cat-23.11/lib/Makefile
@@ -119,10 +119,10 @@ endif
 
 HDR = pqos.h
 PREFIX ?= /usr/local
-LIB_INSTALL_DIR ?= $(PREFIX)/lib
-HDR_DIR ?= $(PREFIX)/include
+LIB_INSTALL_DIR ?= $(DESTDIR)/usr/lib64
+HDR_DIR ?= $(DESTDIR)/usr/include
 DEPFILE = $(LIB).dep
-NOLDCONFIG ?= n
+NOLDCONFIG ?= y
 
 all: $(LIBNAME)
 
Index: intel-cmt-cat-23.11/pqos/Makefile
===================================================================
--- intel-cmt-cat-23.11.orig/pqos/Makefile
+++ intel-cmt-cat-23.11/pqos/Makefile
@@ -94,8 +94,8 @@ MAN = pqos.8
 
 # XXX: modify as desired
 PREFIX ?= /usr/local
-BIN_DIR = $(PREFIX)/bin
-MAN_DIR = $(PREFIX)/man/man8
+BIN_DIR = $(DESTDIR)/usr/sbin
+MAN_DIR = $(DESTDIR)/usr/share/man/man8
 
 SRCS = $(sort $(wildcard *.c))
 OBJS = $(addprefix $(OBJDIR)/,$(SRCS:.c=.o))
Index: intel-cmt-cat-23.11/rdtset/Makefile
===================================================================
--- intel-cmt-cat-23.11.orig/rdtset/Makefile
+++ intel-cmt-cat-23.11/rdtset/Makefile
@@ -95,8 +95,8 @@ MAN = rdtset.8
 
 # XXX: modify as desired
 PREFIX ?= /usr/local
-BIN_DIR = $(PREFIX)/bin
-MAN_DIR = $(PREFIX)/man/man8
+BIN_DIR = $(DESTDIR)/usr/sbin
+MAN_DIR = $(DESTDIR)/usr/share/man/man8
 
 SRCS = $(sort $(wildcard *.c))
 OBJS = $(SRCS:.c=.o)
Index: intel-cmt-cat-23.11/tools/membw/Makefile
===================================================================
--- intel-cmt-cat-23.11.orig/tools/membw/Makefile
+++ intel-cmt-cat-23.11/tools/membw/Makefile
@@ -39,8 +39,8 @@ MAN = membw.8
 
 # XXX: modify as desired
 PREFIX ?= /usr/local
-BIN_DIR = $(PREFIX)/bin
-MAN_DIR = $(PREFIX)/man/man8
+BIN_DIR = $(DESTDIR)/usr/bin
+MAN_DIR = $(DESTDIR)/usr/share/man/man8
 
 CFLAGS=-W -Wall -Wextra -Wstrict-prototypes -Wmissing-prototypes \
 	-Wmissing-declarations -Wold-style-definition -Wpointer-arith \
